packages/bpk-component-nudger/src/bpk-nudger.scss

```diff
@@ -35,12 +35,4 @@
     text-align: center;
     vertical-align: middle;
   }
-
-  &__icon {
-    fill: $bpk-button-secondary-color;
-
-    &--disabled {
-      fill: $bpk-button-disabled-color;
-    }
-  }
```

Comment:
These seem to be redundant as the styles inherited from button are overriding them. I tested in storybook and still seems to work fine.